modules/consensus/consensusset.go

```diff
@@ -100,12 +100,12 @@ func New(gateway modules.Gateway, persistDir string) (*ConsensusSet, error) {
 
 		dosBlocks: make(map[types.BlockID]struct{}),
 
-		clock: types.StdClock{},
-
 		marshaler: encoding.StdGenericMarshaler{},
 
 		blockRuleHelper: stdBlockRuleHelper{},
 
+		blockValidator: NewBlockValidator(),
```

Comment:
> all of these fields I feel should be clustered - marshaler, blockRuleHelper, blockValidator

Fixed: a1741d3

> also, 'RuleHelper' feels like a weak name, but I don't have any good alternate suggestions at the moment.

Yeah, I agree. Any time I have to use the name "Helper" I feel like I'm giving a suboptimal name, but I can't think of a better name. It's not a rule *generator*, not a rule *enforcer*, not a rule *validator*. It generates values that are needed for enforcing a rule, but I don't know a succinct way of describing that.